### PR TITLE
Improves #163: Removed Hard Coded Dimensions

### PR DESCRIPTION
**Pull request by madhavrathi**
Fixes issue #163 

Changes: As requested by @pa1pal  i have removed hard coded dimensions.
@Abhi2424shek  Please Review

### CODE REVIEW

**Comment by madhavrathi** [comment]
@Abhi2424shek Please Review

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 1]
```diff
@@ -73,7 +73,9 @@
 	double lon;
 	int statusScreen = 0;
 	View view;
-
+    //Dimensions for Camera Switch and Flash Button
+    int width = 60;
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Still the static value. That's not the point of adding them pro-grammatically. It should be screen width and height based.
Add two functions in the `Utils` to calculate screen height and screen width. According to them give the value of width and height of icons.

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 2]
```diff
@@ -302,6 +304,9 @@ public void onClick(View v) {
 		});
 		camera_switch = (ImageButton)view.findViewById(R.id.switch_camera);
 		camera_switch.setImageResource(R.drawable.camera_switch);
+        LinearLayout.LayoutParams parmsswitch = new LinearLayout.LayoutParams(width,height);
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Call here the function of Utils and give width and height value

**Comment by madhavrathi** [comment]
@pa1pal @Abhi2424shek  Please Review the changes.

File: app/src/main/java/vn/mbm/phimp/me/Camera2.java [hunk 3]
```diff
@@ -302,6 +303,10 @@ public void onClick(View v) {
 		});
 		camera_switch = (ImageButton)view.findViewById(R.id.switch_camera);
 		camera_switch.setImageResource(R.drawable.camera_switch);
+        LinearLayout.LayoutParams parmsswitch = new LinearLayout.LayoutParams(Utils.getScreenWidth(getContext())/15,
+                Utils.getScreenHeight(getContext())/15);
+        camera_switch.setLayoutParams(parmsswitch);
+        parmsswitch.setMarginStart(20);
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: Properly intend the code.

Comment (madhavrathi) [inline app/src/main/java/vn/mbm/phimp/me/Camera2.java]: @Abhi2424shek  Done

**Comment by madhavrathi** [comment]
@pa1pal @Abhi2424shek Please Review

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -5,10 +5,8 @@
   <dimen name="fab_margin">16dp</dimen>
   <dimen name="headingText">16sp</dimen>
 
-  <dimen name="cameraIcon">30dp</dimen>
-
   <dimen name="image_selector_size">25dp</dimen>
-
+
```
Comment (Abhi2424shek) [inline app/src/main/res/values/dimens.xml]: Unwanted trailing whitespaces. Please remove it.

**Comment by Abhi2424shek** [comment]
Can you pls check the failing tests?

File: upload_apk_forks.sh [hunk 5]
```diff
@@ -14,15 +14,15 @@ cp -R app/build/outputs/apk/app-debug.apk $HOME/buildApk/
 
 # Setting up git
 cd $HOME
-git config --global user.email "noreply@travis.com"
-git config --global user.name "Travis CI"
```
Comment (Abhi2424shek) [inline upload_apk_forks.sh]: Can you pls remove the change and update it from development.

**Comment by madhavrathi** [comment]
@Abhi2424shek  Closing it due to some errors in my fork. Will re-open soon.

**Pull request closed without merging**